package.json

```diff
@@ -11,27 +11,28 @@
     "@babel/plugin-transform-runtime": "^7.10.4",
```

Comment:
@maltheism are the changes to this file necessary ?

Reply:
@ridz1208 likely not. that happened when I was trying to get the new DQT to work. Turns out the htdocs/vender/js/react needed newer versions added.